spec/values/calculation/round/one_argument.hrx

```diff
@@ -0,0 +1,151 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: round(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> positive/input.scss
+a {b: round(1)}
+
+<===> positive/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+a {b: round(-5.6)}
+
+<===> negative/output.css
+a {
+  b: -6;
+}
+
+<===>
+================================================================================
+<===> operation/preserves_units/input.scss
+a {b: round(-7px / 4em) * 1em}
+
+<===> operation/preserves_units/output.css
+a {
+  b: -2px;
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  b: round(var(--c))
+}
+
+<===> preserved/variable/output.css
+a {
+  b: round(var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/interpolation/input.scss
+a {
+  e: round(3px + #{"3"});
```

Comment:
To make it clearer _why_ this case is preserved, consider making it something that produces valid CSS like `round(#{"5.5px, 1px"})`

Reply:
Done, I changed this test and moved it to the two-argument directory since the result contains two arguments.